src/status_im2/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -52,7 +52,7 @@
      [quo/page-nav
       {:background :blur
        :icon-name  :i/arrow-left
-       :on-press   #(rf/dispatch [:navigate-back-within-stack :identifiers])}]
+       :on-press   #(rf/dispatch [:navigate-back-within-stack :new-to-status])}]
```

Comment:
I understand this is probably already discussed with designers etc, but seems odd to jump back to that page from enable_notications 🤔

Reply:
It's the component id, so it would pop to the previous screen in the stack where the component id is located, not to new-to-status. The thing is that probably there is no new-to-status component if in the stack if the user initiates the process from the account selection screen and not from the welcome screen in the onboarding, so we need to check if this works for that case.